third_party/move/move-vm/test-utils/src/gas_schedule.rs

```diff
@@ -39,6 +35,17 @@ use std::{
     u64,
 };
 
+// gas meter for test-utils
+pub trait TestGasMeter: GasMeter {
+    fn instantiate(&self) -> Self;
+    fn remaining_gas(&self) -> Gas;
+    fn charge_write_set(
```

Comment:
Okay, this one is quite interesting. 

Right now `trait GasMeter` does not provide an interface for charging a write set, because that logic is specific to the platform on which the Move VM is being used. In the case of Aptos, this logic belongs to `trait AptosGasMeter`, integral to the transaction execution flow.

Unit testing is unique interesting, in the sense that it does NOT actually run a full transaction, hence lacking the concept of charging for a write set. Nonetheless, it is very reasonable to want to assess the costs of the global operations. (@georgemitenkov I have a feeling that it may require a bit more work to support aggregators properly)

I think overall I'm in favor of keeping this interface, but let's rename it to `fn charge_write_set_for_unit_tests` to avoid confusion.